helm/alfresco-content-services/pre-release_values.yaml

```
@@ -2,44 +2,44 @@
 # the chart
 alfresco-repository:
   image:
-    tag: 23.2.1
+    tag: 23.3.0-A9
```

```suggestion
    tag: 23.3.0-A10
```
Looked at pre released docker compose and spotted difference for this image

---

we need to stick to A9 due to an issue on A10 that needs to be investigated further

but thanks to this I realized that:
* we were not testing pre-release compose
* there was an issue preventing sync service update in pre-release compose
 
😮‍💨